tests/dashbio_demos/app_molecule3d.py

```diff
@@ -23,6 +26,93 @@
 DATAPATH = os.path.join(".", "tests", "dashbio_demos", "sample_data", "molecule3d_")
 
 
+data_info = {
+    '{}4uft.pdb'.format(DATAPATH): {
+        'name': 'Measles Nucleocapsid',
+        'description': dcc.Markdown(s(r'''
+        The measles nucleoprotein forms a large helical complex with
+        RNA... It is thought to chaperone the process of replication and
+        transcription by providing a ready site for binding of the
+        polymerase/phosphoprotein complex while a new RNA chain is being
+        built.
+
+        The structure includes the stable core domain of the
+        nucleoprotein and a strand of RNA, but the flexible tail of
+        nucleoprotein was removed in this study.
+        ''')),
+        'link': 'http://pdb101.rcsb.org/motm/231'
+    },
+
+    '{}1yi5.pdb'.format(DATAPATH): {
+        'name': 'a-cobratoxin-AChBP complex',
+        'description': dcc.Markdown(s(r'''
+
+        The crystal structure of the snake long alpha-neurotoxin,
+        alpha-cobratoxin, bound to the pentameric
+        acetylcholine-binding protein (AChBP) from Lymnaea
+        stagnalis...
+
+        The structure unambiguously reveals the positions and
+        orientations of all five three-fingered toxin molecules
+        inserted at the AChBP subunit interfaces and the
+        conformational changes associated with toxin binding.
+        ''')),
+        'link': 'https://www.rcsb.org/structure/1yi5'
+    },
+
+    '{}1su4.pdb'.format(DATAPATH): {
+        'name': 'Calcium ATPase',
+        'description': dcc.Markdown(s(r'''
+        The calcium pump allows muscles to relax after... \[muscle\]

```

Comment:
What are '...' supposed to mean here? And which rendering does `\[` yield?

Reply:
I'm quoting the website that is specified in "link". The escaped square brackets just yield square brackets in the final Markdown code (since square brackets have a meaning in Markdown already). https://writingcommons.org/open-text/research-methods-methodologies/integrate-evidence/incorporate-evidence/453-inserting-or-altering-words-in-a-direct-quotation

Reply:
Oh, I see, I didn't realize it was a direct quotation. Yes, square brackets need escaping in Markdown. Thanks!